doc/api/stream.md

```diff
@@ -237,9 +237,16 @@ The `pipeline` API provides [callback version][stream-pipeline]:
 
 <!-- YAML
 added: v15.0.0
+changes:
+  - version:
+    - v18.14.0
+    - v20.0.0
```

Comment:
https://github.com/nodejs/node/commit/32254988ba7c0c0a68d8dfbc32430466370bf170 shows it is added starting from `v20.0.0`
https://github.com/nodejs/node/commit/cde59606cd shows it is backported to `v18.14.0`